packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -232,20 +232,24 @@ const handleEnterKeyOnList = (editor) => {
   }
 };
 
+// The max-height is decided togher with the design team
 const Img = styled.img`
+  max-height: 512px;
```

Comment:
I think this height is same as the editor height design wants to set, but now that we have toolbar also inside the editor height, this image is not fully visible. IMO We might need to subtract toolbar height.

So you can fix typo and update comment.

Reply:
We wanted to respect a maximum height like we did for the Markdown Rich text editor, if you have a big screen in the future we want to provide also the possibility to expand the Editor like we did for the Markdown Rich Text Editor